docs/pages/deployment/tls-configuration.rst

```diff
@@ -72,37 +72,25 @@ When using a (level 4) load balancer that does not inspect or alter requests, TL
 .. raw:: html
     :file: ../../_static/images/diagrams/network infrastructure layouts-TLS-Pass-through.svg
 
-This set up does not need additional configuration.
+This setup does not need additional configuration.
 
 Configuration for `HAProxy <https://www.haproxy.com/>`_ could look like this (given the TLS configuration in the previous section):
 
 .. code-block::
 
-    listen grpc
-        bind *:5555
+    default
         mode tcp
-        use_backend nuts_node_grpc
 
-    backend nuts_node_grpc
-        mode tcp
+    listen grpc
+        bind *:5555
         server node1 nodeA-backend:5555 check
 
-    listen public_https
+    listen public
         bind *:443
-        mode tcp
-        use_backend nuts_node_public_https
-
-    backend nuts_node_public_https
-        mode tcp
         server node1 nodeA-backend:443 check
 
-    listen n2n_https
+    listen n2n
         bind *:8443
```

Comment:
mode tcp isn't required?

Reply:
```
    default
         mode tcp
```
sets everything to tcp. But this is not any shorter than just adding `mode tcp` to all listeners, so I will change it back.